modules/using-must-gather.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// TBD
+
+[id="using-must-gather_{context}"]
+= Using Master-Gather
+
+include::modules/common-attributes.adoc[]
+:context: migrating-3-4
+:migrating-3-4:
+
+Must-Gather for MTC is a tool built on top of {product-title}’s standard must-gather tool to gather MTC-specific resources. Therefore, you need to run Must-Gather for MTC  when you migrate containers.
+
+.Prerequisites
+
+* {product-title} 4.x
+
+.Procedure
+
+. Create and navigate to the directory where you want to store the `must-gather` data.
```

Comment:
Is this how OCP docs normally writes this?

Reply:
Usually, they start with "Navigate..." 
https://docs.openshift.com/container-platform/4.2/support/gathering-cluster-data.html

I added "Create and .." based on your comment. Please advise.

Reply:
"Navigate to" is fine with me

Reply:
Fixed.